aip_x1_launch/launch/velodyne_node_container.launch.py

```diff
@@ -22,13 +22,10 @@
 from launch_ros.actions import ComposableNodeContainer
 from launch_ros.actions import LoadComposableNodes
 from launch_ros.descriptions import ComposableNode
-import yaml
 
 
 def get_vehicle_info(context):
-    path = LaunchConfiguration("vehicle_param_file").perform(context)
-    with open(path, "r") as f:
-        p = yaml.safe_load(f)["/**"]["ros__parameters"]
+    p = context.launch_configurations.get("ros_params", {})
     p["vehicle_length"] = p["front_overhang"] + p["wheel_base"] + p["rear_overhang"]
```

Comment:
Is there a problem with pushing it to p[vehicle_length]? because p get from ros_params.

Reply:
I don't think pushing extra parameters to `p` affects `context.launch_configurations`.
```
In [3]: a = {"aaa": "A", "bbb": "B"}

In [4]: x = a.get("aaa")

In [5]: print(x)
A

In [6]: x = "C"

In [7]: print(x)
C

In [8]: print(a)
{'aaa': 'A', 'bbb': 'B'}
```

Reply:
However, many unnecessary items are returned, so I will create a new map for return value.

Reply:
@yukkysaito addressed at https://github.com/tier4/aip_launcher/pull/26/commits/81d272d0d437ff54675fe92488f477e14fcee46c

Reply:
I think it's better this way :+1: